internal/controller/ols_app_server_assets.go

```diff
@@ -116,37 +116,33 @@ func (r *OLSConfigReconciler) generateOLSConfigMap(cr *olsv1alpha1.OLSConfig) (*
 		}
 		providerConfigs = append(providerConfigs, providerConfig)
 	}
-	// TODO: Update DB
-	// redisMaxMemory := intstr.FromString(RedisMaxMemory)
-	// redisMaxMemoryPolicy := RedisMaxMemoryPolicy
-	// redisSecretName := RedisSecretName
-	// redisConfig := cr.Spec.OLSConfig.ConversationCache.Redis
-	// if redisConfig.MaxMemory != nil && redisConfig.MaxMemory.String() != "" {
-	// 	redisMaxMemory = *cr.Spec.OLSConfig.ConversationCache.Redis.MaxMemory
-	// }
-	// if redisConfig.MaxMemoryPolicy != "" {
-	// 	redisMaxMemoryPolicy = cr.Spec.OLSConfig.ConversationCache.Redis.MaxMemoryPolicy
-	// }
-	// if redisConfig.CredentialsSecret != "" {
-	// 	redisSecretName = cr.Spec.OLSConfig.ConversationCache.Redis.CredentialsSecret
-	// }
-	// redisPasswordPath := path.Join(CredentialsMountRoot, redisSecretName, OLSComponentPasswordFileName)
-	// conversationCache := ConversationCacheConfig{
-	// 	Type: string(OLSDefaultCacheType),
-	// 	Redis: RedisCacheConfig{
-	// 		Host:            strings.Join([]string{RedisServiceName, r.Options.Namespace, "svc"}, "."),
-	// 		Port:            RedisServicePort,
-	// 		MaxMemory:       &redisMaxMemory,
-	// 		MaxMemoryPolicy: redisMaxMemoryPolicy,
-	// 		PasswordPath:    redisPasswordPath,
-	// 		CACertPath:      path.Join(OLSAppCertsMountRoot, RedisCertsSecretName, RedisCAVolume, "service-ca.crt"),
-	// 	},
-	// }
 
+	postgresSharedBuffers := intstr.FromString(PostgresSharedBuffers)
```

Comment:
Do you mind to move the entire Postgres configuration to the dedicated function?
Something Like `conversationCache := r.buildConversationCacheConfig(cr)`

This will allow us to make the code a little cleaner and improve code readability